lib/ffmpeg/vaapi/encoder.py

```diff
@@ -32,13 +32,11 @@ def profile(self):
   @property
   def qp(self):
     def inner(qp):
-      if self.codec in [Codec.VP8, Codec.VP9, Codec.AV1]:
-        return f" -global_quality {qp}"
-
+      if self.codec not in [Codec.AVC, Codec.HEVC, Codec.VP8, Codec.VP9, Codec.AV1]:
+        return f" -q {qp}"
```

Comment:
@xhaihao What's the difference between `-q` and `-qp` ffmpeg options?  I see `h264_vaapi` and `h265_vaapi` has `-qp` option.

Reply:
-qp is a private option in h264_vaapi and h265_vaapi encoders however -q is a common output option of FFmpeg tool. We may use the same -q option for all qsv, vaapi, d3d12 encoders.

Reply:
Thanks!

Can you confirm the mapped QP ranges in the tests are still correct?  They still appear different in the test logic for each plugin.  It would be nice if we could move the qp logic to the base class if all plugins were the same.

Reply:
Ideally we should move the qp logic to the base class because QP range is codec related. However, as you know, we have some specific requirement in qsv.
Note -q is set in the command line, CQP mode is used, so -q can't be used to set quality based parameter in other RC modes.